libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -832,6 +832,16 @@ singleton AC_PosControl method get_posvelaccel_offset boolean Vector3f'Null Vect
 singleton AC_PosControl method get_vel_target boolean Vector3f'Null
 singleton AC_PosControl method get_accel_target boolean Vector3f'Null
 
+include AC_Sprayer/AC_Sprayer.h
+singleton AC_Sprayer depends HAL_SPRAYER_ENABLED
+singleton AC_Sprayer scheduler-semaphore
+singleton AC_Sprayer rename sprayer
+singleton AC_Sprayer method run void boolean
+singleton AC_Sprayer method running boolean
```

Comment:
These are just terrible names.  Adding them to bindings makes these harder to change later.

Would `set_spraying_allowed()` and `spraying_is_allowed()`  be accurate here?